tests/test_modeling_lxmert.py

```diff
@@ -726,3 +727,24 @@ def test_retain_grad_hidden_states_attentions(self):
         self.assertIsNotNone(attentions_vision.grad)
         self.assertIsNotNone(hidden_states_vision.grad)
         self.assertIsNotNone(attentions_vision.grad)
+
+
+@require_torch
+class LxmertModelIntegrationTest(unittest.TestCase):
+    @slow
+    def test_inference_no_head_absolute_embedding(self):
+        model = LxmertModel.from_pretrained(LXMERT_PRETRAINED_MODEL_ARCHIVE_LIST[0])
+        input_ids = torch.tensor([[101, 345, 232, 328, 740, 140, 1695, 69, 6078, 1588, 102]])
+        num_visual_features = 10
+        _, visual_feats = np.random.seed(0), np.random.rand(1, num_visual_features, LxmertModel.config.visual_feat_dim)
```

Comment:
This should not be able to work, the `LxmertModel` has no attribute config. You probably meant `model` instead!

Reply:
yes, u r right!